feature/interface/aggregate/otg_tests/balancing_test/balancing_test.go

```diff
@@ -260,14 +260,18 @@ func (tc *testCase) verifyLAG(t *testing.T) {
 		t.Logf("Waiting LAG OTG ports to start collecting and distributing")
 		for _, p := range tc.atePorts[1:] {
 			_, ok := gnmi.Watch(t, tc.ate.OTG(), gnmi.OTG().Lacp().LagMember(p.ID()).Collecting().State(), time.Minute, func(val *ygnmi.Value[bool]) bool {
+				time.Sleep(1 * time.Second)
```

Comment:
why is this here?

Await is edge triggered - waiting a second shouldn't provide any value